rules/naming/tests/Rector/Assign/RenameVariableToMatchGetMethodNameRector/Fixture/arrow_function.php.inc

```diff
@@ -0,0 +1,19 @@
+<?php
+
+function testArrowFunction() {
```

Comment:
Better always use current namespace in the fixture files.

It makes classs/functions unique and it prevents somebody else using the same `function test()` in another fixture file.